src/test/java/pages/DynamicControlsPage.java

```diff
@@ -0,0 +1,81 @@
+package pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.NoSuchElementException;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+
+
+public class DynamicControlsPage extends BasePage {
+    public static final String DYNAMIC_CONTROLS_URL = "http://the-internet.herokuapp.com/dynamic_controls";
+    public final By CHECKBOX_ELEMENT = By.id("checkbox");
+    public final By BUTTON_ENABLE = By.xpath("/html/body/div[2]/div/div[1]/form[2]/button");
+    public final By FIELD_INPUT = By.xpath("//*[@id='input-example']/input");
+    public final By BUTTON_REMOVE = By.xpath("//*[@id='checkbox-example']/button");
+    public final By TEXT_AFTER_CLICK_ENABLE = By.xpath("/html/body/div[2]/div/div[1]/form[2]/p");
+    public final By TEXT_AFTER_BUTTON = By.id("message");
+    public final boolean EXP_RESULT_CHECKBOX = false;
+    public final boolean EXP_RESULT_INPUTFIELD = true;
+    public final String EXP_RESULT_MESSAGE_CHECKBOX = "It's gone!";
+    public final String EXP_RESULT_MESSAGE_INPUTFIELD = "It's enabled!";
+
+    public DynamicControlsPage(WebDriver driver) {
+        super(driver);
+    }
+    /*
+    Open website http://the-internet.herokuapp.com/dynamic_controls
+     */
+    @Override
+    public void openPage() {
+        driver.get(DYNAMIC_CONTROLS_URL);
+
+    }
+    /*
+    Checking checkbox
+     */
+    public boolean isCheckbox() {
+        try {
+            driver.findElement(CHECKBOX_ELEMENT).isDisplayed();
+            return true;
+        } catch (NoSuchElementException e) {
+            return false;
+        }
+    }
+    /*
+    Find element on the path
+     */
+    public WebElement findElementForTest(By element) {
```

Comment:
bad approach, don't pass locators in Test classes like you did. Make separate methods for click on different locators.

Reply:
done